tests/datetime/test_datetime.py

```diff
@@ -44,16 +44,15 @@ def test_can_find_before_the_year_2000():
 
 @checks_deprecated_behaviour
 def test_can_find_each_month():
-    for i in hrange(1, 12):
-        minimal(datetimes(), lambda x: x.month == i)
+    for i in hrange(12):
+        datetimes().filter(lambda x: x.month == i + 1).example()
```

Comment:
Mmm. If these tests were’t deprecated, I’d be tempted to put them under a `@pytest.mark.parametrize` decorator, parametrised by strategy. Do we have any repetition like this in the new tests?